pkg/codegen/python/doc.go

```diff
@@ -39,7 +39,16 @@ func (d DocLanguageHelper) GetDocLinkForPulumiType(pkg *schema.Package, typeName
 
 // GetDocLinkForResourceType is not implemented at this time for Python.
```

Comment:
Let's update the comment now that this function actually does something.

Reply:
done. 😄